site/content/entry/citra-progress-report-2017-july.md

```diff
@@ -0,0 +1,62 @@
++++
+date = "2017-08-01T12:38:00-04:00"
+title = "Citra Progress Report - 2017 July"
+tags = [ "progress-report" ]
+author = "Saphiresurf"
+forum = 0000
++++
+
+With June bringing in many huge improvements to GPU functionality, July has come in to sweep up and improve accuracy in a few distinct areas of Citra. Changes and fixes ranging from texture handling all the way to aesthetic changes that help improve the visual appeal and customizability of Citra's QT interface. Now, let's get into what's changed!
```

Comment:
"Changes and fixes ranging from texture handling all the way to aesthetic changes that help improve the visual appeal and customizability of Citra's QT interface." -> Is a fragment sentence